### PR TITLE
feat: soft nonce wallets

### PR DESCRIPTION
**Pull request by guru-web3**

### CODE REVIEW

File: src/mpcCoreKit.ts [hunk 1]
```diff
@@ -86,6 +86,8 @@ export class Web3AuthMPCCoreKit implements ICoreKit {
 
   private enableLogging = false;
 
+  private accountIndex;
```
Comment (ieow) [inline src/mpcCoreKit.ts:89]: Can we try move this to the coreKit's state?
so that refresh will retain the previous account index set

**Comment by ieow** [comment]
https://github.com/tkey/tkey-mpc/pull/6
need this tkey pr

**Comment by himanshuchawla009** [comment]
@ieow do we need this PR anymore?

**Comment by himanshuchawla009** [comment]
not required anymore

**Pull request closed without merging**